### DISCUSSION TITLE
Add header only for a single url

### DISCUSSION THREAD

**Discussion by Amaury4sg** [OpenIDC/mod_auth_openidc#874] [Q&A] [answered]
Hello I have a use case I didn't found in the documentation

I wish to protect myapplication.com/*
requiring a valid OIDC token
But we wish to add the headers only on a specific url like myapplication.com/login.html for example
on this url the app behind will create the session cookie and don't need the headers anymore.
When the session on the app is not valid anymore the app will redirect to myapplication.com/login.html

I believe I can remove all the headers with https://httpd.apache.org/docs/current/mod/mod_headers.html
Is there another way with mod_auth_openidc ?

I'm open to suggestions
Thanks for your help (If I missed a similar discussion apologies, a lot of those have the url & headers keyword :))

**Comment by zandbelt** [comment, marked as answer]
you should be able to use [`OIDCPassClaimsAs 
 none`](https://github.com/zmartzone/mod_auth_openidc/blob/v2.4.11.2/auth_openidc.conf#L768-L781) in those Location contexts that should not receive claims in headers downstream

**Reply by Amaury4sg** [reply]
Ok I'll try that thanks for your time !
